app-text/cherrytree/cherrytree-0.38.4.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 )
+PYTHON_REQ_USE="sqlite"
+DISTUTILS_SINGLE_IMPL=true
+
+inherit xdg distutils-r1
+
+DESCRIPTION='A hierarchical note taking application'
+HOMEPAGE='https://www.giuspen.com/cherrytree'
+LICENSE='GPL-3'
+
+SLOT='0'
+SRC_URI="https://github.com/giuspen/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS='~amd64 ~x86'
+IUSE='nls'
+
+RDEPEND="
+	x11-libs/libX11
+	dev-python/dbus-python[${PYTHON_USEDEP}]
+	dev-python/pyenchant[${PYTHON_USEDEP}]
+	>=dev-python/pygtk-2.16:2[${PYTHON_USEDEP}]
+	dev-python/pygtksourceview:2[${PYTHON_USEDEP}]
+"
+DEPEND="${RDEPEND}
+	nls? ( sys-devel/gettext )
+"
+
+PLOCALES='cs de es fr hy it ja lt nl pl pt_BR ru sl tr uk zh_CN'
```

Comment:
why is this in the middle? move this straight after `EAPI` defenition

Reply:
Actually, please remove the whole l10n+nls logic. We don't do that stuff for Python apps.